### PR TITLE
Buddhist-view-fortune-telling-Sheng-yen

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - "https://drive.google.com/file/d/1B_12mi5l5UkduqmuplkCCuKd1fN4nppu/view?usp=drive_link"
+course: karma
+tags:
+  - chan
+  - mind
+  - iddhi
+address: Taipei
+year: 1994
+publisher: Dharma Drum Publishing
+series: Great Dharma Drum
```
Comment (buddhist-uni) [inline _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md:16]: There's a slug for this series! `gdd`

File: _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: "How does Buddhism view the practice of fortune telling?"
+authors:
+  - sheng-yen
+external_url: "https://www.youtube.com/watch?v=kOrIWq0YgNw"
+drive_links:
+  - "https://drive.google.com/file/d/1B_12mi5l5UkduqmuplkCCuKd1fN4nppu/view?usp=drive_link"
+course: karma
+tags:
+  - chan
```
Comment (buddhist-uni) [inline _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md:10]: Did you mean `east-asian`?

File: _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - mind
```
Comment (buddhist-uni) [inline _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md:11]: maybe `problems` instead of mind?

**Review by buddhist-uni — COMMENTED** [review]
You're getting good at these! 🤠

File: _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md (hunk 3 above)
Comment (JJM317) [reply _content/av/buddisism-view-fortune-telling-fengshui_sheng-yen.md:11]: problems is cool. You'll have to explain to me once more what the mind tag is generally used for. In this short video, Sheng-yen spends time talking about cultivating the mind, etc.

**Comment by JJM317** [comment]
Hi Khemarato,

Can we meet at 8pm (Thai time) on Monday, instead of our usual 7pm? If not,
I'll have to push our meeting out by two weeks.

Thanks for whatever you can do.

~Joe

**Pull request merged**